apps/daimo-mobile/src/view/screen/HomeScreen.tsx

```diff
@@ -158,7 +180,11 @@ function HomeScreenInner({ account }: { account: Account }) {
         scrollsToTop={false}
         scrollEnabled={searchPrefix == null && !isBottomSheetOpen}
         refreshControl={
-          <RefreshControl refreshing={refreshing} onRefresh={onRefresh} />
+          <AnimatedRefreshControl
+            refreshing={refreshing}
+            onRefresh={onRefresh}
+            style={refreshAnimatedStyle}
+          />
```

Comment:
hmm, the fade in and out feels a bit wonky. maybe change change text to `Retrying...` and then back to `Offline` instead?